akka-operator/values.yaml

```diff
@@ -38,3 +38,5 @@ resources:
 nodeSelector: {}
 tolerations: []
 affinity: {}
+cloudProvider:
+  name: "gcp"
```

Comment:
you can keep it like this for testing, but, very likely, it's not the default we want to use. (e.g. AWS is already complete)

Reply:
I was thinking we can do this and override during `helm install` using `--set` to pass in the proper value

Reply:
yup! with two caveats:
 - we have to decide on a proper default
 - this parameter should switch the Docker image repository as well
 Ping me if you need any help 👍

Reply:
Good point! Docker image repository is in `values` (https://github.com/lightbend/akka-operator-helm/blob/master/akka-operator/values.yaml#L6) - hence can't we override it with `--set` in `helm install` ?

Reply:
I was thinking about having something on those lines (not validated with Patrik's or Ray's view though).

provider:
 - none - `default`
 - aws
 - gcp

`imageRepository`: blank - default

imageRepository selected with something like:
```
if (gcp && imageRepository == blank) {
  use default gcpRepo
} else if (aws && imageRepository == blank) {
  use default awsRepo
} else {
  use imageRepository (e.g. will fail if blank)
}
```

Reply:
matches exactly with my thoughts @andreaTP .. having `none` and blank as defaults and using `--set` during `helm install`

Reply:
sounds good to have one cloudProvider property like that and derive other values from it

Reply:
yep, sound good to me too.

Reply:
👍 to the default being none